prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Jae Song';
+    console.log(fullName);
+    var numberOfPets = '0';
+    console.log(numberOfPets);
+    var likesVegetables = Boolean(true);
+    console.log(likesVegetables);
+    var nothing = 'null';
```

Comment:
Hi Jae, be sure to leave the quotation marks off of your number and null values so that the JavaScript engine doesn't see them as strings.

Reply:
![prepjavavariables2](https://user-images.githubusercontent.com/99282675/156279865-03917e84-66fd-404a-ae7c-89e93917f00c.png)